src/aliyun/credential.rs

```diff
@@ -138,8 +138,11 @@ impl Loader {
         let role_session_name = &self.config.role_session_name;
 
         // Construct request to Aliyun STS Service.
-        let url = format!("https://sts.aliyuncs.com/?Action=AssumeRoleWithOIDC&OIDCProviderArn={}&RoleArn={}&RoleSessionName={}&Format=JSON&Version=2015-04-01&Timestamp={}&OIDCToken={}", provider_arn, role_arn, role_session_name, format_rfc3339(now()), token);
 
+        let url = match &self.config.sts_endpoint{
```

Comment:
Please create the STS endpoint first, then add queues instead of repeating the long string twice.

It's better to have an `get_sts_endpoint` function so we can add regional sts in the future.